go/cmd/pfacct/radius.go

```diff
@@ -582,7 +582,9 @@ type RadiusStatements struct {
 func setupStmt(db *sql.DB, stmt **sql.Stmt, sql string) {
 	var err error
 	if *stmt, err = db.Prepare(sql); err != nil {
-		panic(err)
+		log.Logger().Error(fmt.Sprintf("Failed to prepare statement '%s': %s", sql, err))
+		time.Sleep(5 * time.Second)
+		go setupStmt(db, stmt, sql)
```

Comment:
Maybe a better approach would be to use a WaitGroup in RadiusStatements::Setup.
So we can wait until all of the statements succeed before moving forward.

Reply:
Doing this would prevent pfacct from starting and accepting requests until this completes which is not desirable. We should be able to startup in R/O mode and the `INSERT/UPDATE` statements can't be prepared in this mode when going through ProxySQL

Reply:
We would need to detect If the statement is nil.
When starting pfacct in readonly mode, what behavior did you see?

Reply:
If we leave it with the previous way it was, it would make pfacct crash on startup when in read-only mode. With this fix, it starts but doesn't respond well (which is expected) and then when the read-only goes away, it starts processing packets

Reply:
I updated the setupStmt function

Reply:
Reviewed that change, I'm happy with it.

If all is good for you too now, let's merge this baby!